### ISSUE TITLE
Duplicate IDs on ONS logo SVG icons used in header masthead and footer (WCAG Level A fail)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2234]
Need to set a param for the  `id` used in the `<title>` element for ons icons set in `onsIcon()`, so we can specify unique IDs in the header and footer.

---

Issue ID: DAC_Non_Unique_Ids_02
URL: https://ons-design-system.netlify.app/patterns/download-resources/examples/download-resources/
Page title: Service name
Journey: Journey 2, Step 2

The office for national statistics logo images on the page contains an id attribute. The value of the id attribute has been duplicated. This could cause complications for assistive technology reading the code. It may also cause a mismatch of content if the id is used multiple times.

### Solution:
Id attribute values must be unique. Ensure that the office for national statistics logo images is revised to have unique id attribute values.

**Comment by jrbarnes9** [comment]
@yatesn  from page 18 of the June 2022 DAC DS report.
